doc/classes/BaseMaterial3D.xml

```diff
@@ -215,10 +215,11 @@
 			If [code]true[/code], the object is rendered at the same size regardless of distance.
 		</member>
 		<member name="grow" type="bool" setter="set_grow_enabled" getter="is_grow_enabled" default="false">
-			If [code]true[/code], enables the vertex grow setting. See [member grow_amount].
+			If [code]true[/code], enables the vertex grow setting. This can be used to create mesh-based outlines using a second material pass and its [member cull_mode] set to [constant CULL_FRONT]. See also [member grow_amount].
+			[b]Note[/b] Vertex growth cannot create new vertices, which means that visible gaps may occur in sharp corners. This can be alleviated by designing the mesh to use smooth normals exclusively using [url=https://wiki.polycount.com/wiki/Face_weighted_normals]face weighted normals[/url] in the 3D authoring software. In this case, grow will be able to join every outline together just like in the original mesh.
```

Comment:
```suggestion
			[b]Note:[/b] Vertex growth cannot create new vertices, which means that visible gaps may occur in sharp corners. This can be alleviated by designing the mesh to use smooth normals exclusively using [url=https://wiki.polycount.com/wiki/Face_weighted_normals]face weighted normals[/url] in the 3D authoring software. In this case, grow will be able to join every outline together, just like in the original mesh.
```

Reply:
Here's the softwares that you can use free. You guys just visit our site and get all the latest and older softwares. Kindly click on here and visit our website and read more.
https://bit.ly/3YAnttr